### PR TITLE
Add example for converting to float32 numpy array

### LINKED ISSUE #289: change Audiosegment to ndarray for librosa to add effect
I wonder how to change Audiosegment to float32 ndarray for librosa to manipulate.

### PR DESCRIPTION
**Pull request by cghawthorne**
This question comes up frequently enough that I thought it would
be useful to have an "official" example people can copy.

Fixes #289.

### CODE REVIEW

**Comment by jiaaro** [comment]
sorry for the delay, this is a good addition - would you mind adding the converstion back to an audiosegment (which is usually the next question people ask :) )

**Comment by cghawthorne** [comment]
Thanks! Added support for multi-channel to the example and showed how to use scipy.io.wavfile.write to convert back to AudioSegment. Let me know if you'd like any other changes!

File: API.markdown [hunk 1]
```diff
@@ -553,6 +553,32 @@ shifted_samples_array = array.array(sound.array_type, shifted_samples)
...
+
+And how to convert it back to an AudioSegment:
+
+```python
+import io
+import scipy.io.wavfile
+
+wav_io = io.BytesIO()
+scipy.io.wavfile.write(wav_io, 16000, fp_arr)
+sound = pydub.AudioSegment.from_wav(io.BytesIO(wav_io.getvalue()))
```
Comment (jiaaro) [inline API.markdown:579]: can't you use `wav_io.seek(0)` and pass the original `wav_io` to `AudioSegment.from_wav` ?

File: API.markdown [hunk 2]
```diff
@@ -553,6 +553,32 @@ shifted_samples_array = array.array(sound.array_type, shifted_samples)
...
 ```
 
+Here's how to convert to a numpy float32 array:
+
+```python
+import numpy as np
+from pydub import AudioSegment
+
+sound = AudioSegment.from_file("sound1.wav")
+sound = sound.set_frame_rate(1600)
```
Comment (jiaaro) [inline API.markdown:563]: should the frame rate be `16000`?

Comment (cghawthorne) [reply API.markdown]: Oops, fixed!

File: API.markdown (hunk 1 above)
Comment (cghawthorne) [reply API.markdown]: Yes, verified that works and updated the example.

**Pull request merged**